src/AddTeamMemberForm.jsx

```diff
@@ -14,21 +14,30 @@ class AddTeamMemberForm extends React.Component {
   }
   onChange(key) {
     // Return a curried function that sets a value to the key in this.state
-    return function(value) {
+    return (value) => {
```

Comment:
I'm not requesting that you change this, but I just wanted to note that I believe if you have an anonymous function with only one parameter, you do not need parenthesis, e.g. you could have written:
```javascript
return value => {
  /* ... */
}
```
instead of
```javascript
return (value) => {
  /* ... */
}
```